### PR TITLE
Fix wrong configuration docs

### PR DESCRIPTION
**Pull request by soq**
## Description
https://github.com/spotify/luigi/issues/2133

## Motivation and Context
Wrong docs

I didn't test it. If you can give me instruction for running tests in repo on my local machine, it would be nice.

### CODE REVIEW

File: doc/central_scheduler.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ To run the server as a daemon run:
 
 .. code-block:: console
 
-    $ luigid --background --pidfile <PATH_TO_PIDFILE> --logdir <PATH_TO_LOGDIR> --state-path <PATH_TO_STATEFILE>
+    $ luigid --background --pidfile <PATH_TO_PIDFILE> --logdir <PATH_TO_LOGDIR> --state_path <PATH_TO_STATEFILE>
```
Comment (Tarrasch) [inline doc/central_scheduler.rst]: are you sure about this one?? I think  it should be `-` as that's what use for all other command line parameters

File: luigi/cmdline.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ def luigid(argv=sys.argv[1:]):
     parser.add_argument(u'--background', help=u'Run in background mode', action='store_true')
     parser.add_argument(u'--pidfile', help=u'Write pidfile')
     parser.add_argument(u'--logdir', help=u'log directory')
-    parser.add_argument(u'--state-path', help=u'Pickled state file')
+    parser.add_argument(u'--state_path', help=u'Pickled state file')
```
Comment (Tarrasch) [inline luigi/cmdline.py]: Let's aim to change documentation not code. It looks like this change directly contradicts the line I first commented on.

**Comment by Tarrasch** [comment]
How's it going?

> I didn't test it. If you can give me instruction for running tests in repo on my local machine, it would be nice.

Oh yea, see https://github.com/spotify/luigi/blob/master/CONTRIBUTING.rst

**Comment by dlstadther** [comment]
@soq Any chance you can address @Tarrasch 's comments?

File: doc/central_scheduler.rst (hunk 1 above)
Comment (soq) [reply doc/central_scheduler.rst]: Yes, my bad. There should be -.

**Comment by soq** [comment]
@dlstadther Yes, sorry for delay.

**Comment by Tarrasch** [comment]
Thanks @soq I wanted this for a long time. :)

**Pull request merged**